Core/src/main/resources/lang/messages_en.json

```diff
@@ -322,6 +322,8 @@
   "debugsavetest.starting": "<prefix><gold>Starting debugsavetest.</gold>",
   "debugsavetest.done": "<prefix><gold>Database sync finished.</gold>",
   "purge.purge_success": "<prefix><dark_aqua>Successfully purged <amount> plots.</dark_aqua>",
+  "purge.confirm_purge_unknown_bg_enabled": "<prefix><gray>Background UUID caching is enabled. Unknown plot owners may be found using this!</gray>",
+  "purge.confirm_purge_unknown_bg_disabled": "<prefix><gray>Background UUID caching is disabled. Unknown plot owners may be found using this! Enabled in settings.yml</gray>",
```

Comment:
Should this say `Enable` instead of `Enabled`? Sounds quite confusing otherwise

Reply:
oops